src/sinks/humio_logs.rs

```diff
@@ -18,6 +18,10 @@ pub struct HumioLogsConfig {
         default
     )]
     encoding: EncodingConfigWithDefault<Encoding>,
+
+    #[serde(rename = "type")]
```

Comment:
Using `type` here rather than `sourcetype` as this matches Humio's terminology. I couldn't call it `type` in Rust so I just decided to use `sourcetype` internally since it is used with the HEC sink.

https://docs.humio.com/foundational-concepts/events/#type